README.md

```diff
@@ -6,9 +6,11 @@
 
 Welcome to [Yallyo](https://yallyo.com), a language practice platform designed to connect people worldwide for language exchange and conversation practice. This README provides an overview of the project, its features, and how you can get involved as a contributor.
 
-![ Screenshot1](https://yallyo.com/images/yallyo-mobile.png)
-![ Screenshot2](https://yallyo.com/images/yallyo-ipad.png)
-![ Screenshot3](https://yallyo.com/images/yallyo-desktop.png)
+<div style="float: left;">
```

Comment:
bro i didn't know we could add html directly to markdown like this. beyond awesome!